ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js

```diff
@@ -68,11 +77,23 @@ DefaultStory.storyName = 'Default';
 DefaultStory.args = {
   suggestedAssets: [...mockSuggestedAssets],
   tokens: [],
+  symbol: 'META',
+  image: 'metamark.svg',
```

Comment:
I would remove these and move them to the story config so they are available for all stories
```suggestion
```